docker/dev/docker-compose.yml

```diff
@@ -18,14 +18,21 @@ services:
     # actual and only port facing any connection from outside 
       - "1236:1236"
     volumes:
-      - ./nginx-dev.conf:/etc/nginx/nginx.conf:ro
+      - ./nginx.conf:/etc/nginx/nginx.conf:ro
     restart: always
     depends_on:
       - pictrs
       - lemmy-ui
 
   lemmy:
     image: dessalines/lemmy:dev
+    # use this to build your local lemmy server image for development
+    # checkout git repo to build and
+    # run docker compose up --build
+    #build: 
+    #  context: ./build/lemmy
+    #  dockerfile: ./build/lemmy/docker/dev/Dockerfile   
```

Comment:
Its not necessary to clone repo again, it should work with current repo and path more or less like:
```
    build: 
      context: ../..
      dockerfile: Dockerfile
```